README.md

```diff
@@ -3,9 +3,10 @@
 ## Run this demo in your localhost
 
 1. Clone this Repo
-2. Do `yarn install`
-3. Do `yarn start`
-4. Navigate to [http://127.0.0.1:9000](http://127.0.0.1:9000)
+2. Get access token for users registered with Webex Calling. If you don't have an account request for a sandbox - https://developer.webex.com/docs/developer-sandbox-guide
+3. In "js/app.js", put the access tokens (line 126, 128)
+4. Run `yarn start`
+5. Navigate to [http://127.0.0.1:9000](http://127.0.0.1:9000)
```

Comment:
you need to specify here that mytrips.html is one page and other one is agent.html

Reply:
Added.